share/gpodder/extensions/subscription_stats.py

```diff
@@ -66,27 +67,21 @@ def add_page(self, notebook, category, channels):
         channelcell.set_property('alignment', Pango.Alignment.LEFT)
         channelcell.set_property('ellipsize', Pango.EllipsizeMode.END)
         channelcolumn = Gtk.TreeViewColumn(_('Podcast'))
-        channelcolumn.set_sort_column_id(1)
+        channelcolumn.set_sort_column_id(2)
         channelcolumn.pack_start(channelcell, True)
-        channelcolumn.add_attribute(channelcell, 'text', 1)
+        channelcolumn.add_attribute(channelcell, 'text', 2)
         channelcolumn.set_expand(True)
         tree.append_column(channelcolumn)
 
         lastcell = Gtk.CellRendererText()
         lastcell.set_property('xalign', 0)
         lastcell.set_property('alignment', Pango.Alignment.LEFT)
         lastcolumn = Gtk.TreeViewColumn(_('Last Updated'))
-        lastcolumn.set_sort_column_id(3)
+        lastcolumn.set_sort_column_id(4)
         lastcolumn.pack_start(lastcell, True)
-        lastcolumn.add_attribute(lastcell, 'text', 2)
+        lastcolumn.add_attribute(lastcell, 'text', 3)
         tree.append_column(lastcolumn)
```

Comment:
The 'Last Updated' column is too narrow when the dialog is first shown and does not show the full date. Resizing the dialog makes it bounce to the correct width.

Adding `lastcolumn.set_min_width(130)` fixes this somehow, but it's a bit of a hack. If you can find out how to make this column to have the correct width from the start, that would be great.

Reply:
I have no solution for this problem,  I've allowed the column to expand and this solve the problem when the dialog is show, but if the dialog is resized the column is resized also ...

Reply:
Putting the 'Channel' column last and removing `lastcolumn.set_expand(True)` seems to work fine. The channel column then receives all the extra space. Its strings are not ellipsized when the dialog is first rendered, but I think we can live with that.

Reply:
That is a good idea, I've made some more tests and I think maybe the culprit is the  ellipsize, if I comment that line the Updated column is ok, but in the other hand the dialogue  doesn't resize to the initial size!
No solution for this so, I follow your advice and changed the order of the columns

Reply:
Would moving the updated column far left also work? The wide date pulls the days column further from the podcast name, requiring more tracing across to associate them.